src/ScylloClient.ts

```diff
@@ -45,11 +45,17 @@ export class ScylloClient<TableMap extends Tables> {
         this.debug = options.debug || false;
         this.log = options.log || console.log;
     }
-
+    /**
+     * Connect
+     * to the client.
+     */
```

Comment:
I do have to mention that technically the `cassandra-driver` library we are using underneath automatically connects to the servers. So technically this only really holds until that connection is established, but technically this doesnt "trigger a connet" more so then "wait till connection is done"

Reply:
Could you find a way to sort-of clarify that? Cuz I have no clue